src/tcgwars/logic/impl/gen3/PowerKeepers.groovy

```diff
@@ -421,13 +435,11 @@ public enum PowerKeepers implements LogicCardInfo {
             assert my.deck : "Deck is empty"
             powerUsed()
 
-            my.deck.search(max: 1, "Search for a [P] Energy card to attach to one of your Pokemon.", {
-              Card card->card.cardTypes.contains(BASIC_ENERGY) && card.asEnergyCard().containsTypePlain(P)
-              }).each {
-                def tar = my.all.select("Attach $it to? That Pokemon will receive 2 damage counters.")
-                attachEnergy(tar, it)
-                directDamage 20, tar
-              }
+            my.deck.search(max: 1, "Search for a [P] Energy card to attach to one of your Pokemon.", {Card card -> card.asEnergyCard().containsTypePlain(L)}).each {
```

Comment:
Why is this looking for Lightning type instead of Psychic type now?

Reply:
tfw thought I'd tested this card earlier. Will fix in #877, thanks!